numba/unicode.py

```diff
@@ -696,6 +696,66 @@ def zfill_impl(string, width):
     return zfill_impl
 
 
+@register_jitable
+def strip_impl(func_id, string, chars=' '):
+
+    if chars is None:
+        chars = ' '
+
+    str_len = len(string)
+
+    left_it = 0
+    right_it = str_len
+
+    if func_id != 1:
```

Comment:
Here, I would advocate for some global, const style values, and checking for a positive match, something like:

```
if func_id in [LSTRIP, STRIP]:
    ...
```

Reply:
@esc I tried to address this. If you meant something else please let me know. Perhaps I need to implement this http://code.activestate.com/recipes/65207-constants-in-python/?in=user-97991
If you require me to implement `class _const` as described in the link - I would propose to make it in some common place (please recommend a file).

Reply:
No, this is what I had in mind.